New version: MinkowskiFunctionals v0.14.0

JuliaRegistrator opened the pull request:
- Registering package: MinkowskiFunctionals
- Repository: https://github.com/markuspirke/MinkowskiFunctionals.jl
- Created by: @markuspirke
- Version: v0.14.0
- Commit: e8052985b47045e859850db19340a622a263596b
- Reviewed by: @markuspirke
- Reference: https://github.com/markuspirke/MinkowskiFunctionals.jl/commit/e8052985b47045e859850db19340a622a263596b#commitcomment-162189924
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
## Breaking changes

- performance update
- This might no longer work Minkowski functionals on boolean arrays may no longer work
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd66bd200d0b5c32b65d53dbe1345aaf456438055e68e7c153457531d57eee5abe01ff19c1f2fcf7bd654a8727ec3d7c22ba6c50267f889193f6aa6e035ec64f14023b3d415c7723440ef8a82540221fd51848922e6fa4e00a35e31eb67feab11fc86fd3189afb9ffa54019c21bcf0645e37ed7c2bc4ebb52bb7e510fa024ab12b047e2471462e23ff7e9aa57b26ae9a09e6471a82e60baf0559b47d06d2df3c8fe40aea65475dbe8250f98f44b0584cf4da15c1dc117c8bd35112506d175d84160 -->

markuspirke commented:
@JuliaRegistrator register

Release notes:

## Breaking changes

- performance update
- This might no longer work Minkowski functionals on boolean arrays may no longer work

JuliaRegistrator committed:
New version: MinkowskiFunctionals v0.14.0

UUID: 966f3356-8dad-4c1b-950e-472e4a926962
Repo: https://github.com/markuspirke/MinkowskiFunctionals.jl.git
Tree: a7d466f0da2644e06094891f6ace58c6517d7367

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
